cmd/buck/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/textileio/textile/api/common"
+	"github.com/textileio/textile/cmd"
+	buck "github.com/textileio/textile/cmd/buck/cli"
+)
+
+const defaultTarget = "127.0.0.1:3006"
+
+var clients *cmd.Clients
+
+func init() {
+	cobra.OnInitialize(cmd.InitConfig(buck.Config()))
+	buck.Init(rootCmd)
```

Comment:
`buck` uses it's own rootCmd to "init" all of the bucket sub-commands, while `hub` uses an additional `buck` sub-command to suck up `buck push` etc.